React/tic-tac/.babelrc

```diff
@@ -13,7 +13,8 @@
           "@utils": "./src/utils",
           "@config": "./src/config",
           "@services": "./src/services",
-          "@login": "./src/screens/Login"
+          "@login": "./src/screens/Login",
```

Comment:
You don't need to define these aliases. Try to set only aliases for shared folders, but not for a single folder. 🙂